content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,386 @@
+
+> Please note that we are _not_ giving legal advice. This article is simply meant for educational purposes for individuals. Consult legal authorities for the appropriate jurisdiction
+
+
+Accessibility isn't a pure science, however. If you're sighted, this may be an abstract idea at first. However, think of it like this: the colors an app uses or a button's visual placement may convey different messages and meanings depending on their context. This same problem applies to users of screen-readers and other accessible tech as well, just with different constraints. If the screen is visually cluttered, the content may be more difficult to read. Likewise, different accessibility methods will lead to different experiences for users of assistive technology. In both of these scenarios, there may not be objectively correct answers - some may prefer a button placed visually to the left, while others might advocate for it on the right. Similarly, how something is read using a screen reader may make sense to some, but might be confusingly expressed to others.
+
+# Sensible Standards {#wcag}
+
+While accessibility has some levels of subjectivity involved, it's important to note that there _are_ standards surrounding web application's accessibility support. ["Web Content Accessibility Guidelines"](https://www.w3.org/WAI/) (shortened to "WCAG") are guidelines to follow when considering your app's accessibility.  These guidelines are published by a subgroup of the [World Wide Web Consortium](https://www.w3.org/) (shortened to "W3C"), the main international standards organization for the Internet. WCAG acts as the de-facto standard for accessibility guidelines.
+
+There are different scales of accessibility as well. [WCAG includes three different levels of conformance](https://www.w3.org/WAI/WCAG2AA-Conformance):
+
+> - Level A is the minimum level.
+> - Level AA includes all Level A and AA requirements. Many organizations strive to meet Level AA.
+> - Level AAA includes all Level A, AA, and AAA requirements.
+
+Meeting AA requirements is typically seen as a good commitment to accessibility, but AAA will open more doors to your users and is the gold standard for accessible user experience. 
+
+Far from a comprehensive list, A requires:
+
+- [Non-text content to have alternative text](https://www.w3.org/TR/WCAG21/#non-text-content)
+- [All content accessible with keyboard](https://www.w3.org/TR/WCAG21/#keyboard)
```

Comment:
Why did you choose to not have Pause Stop Hide? :(

Reply:
Legit no idea 😅😅😅 I was addressing the article feedback at 3AM and was bouncing back-and-forth between like two dozen things. I must've gotten my wires crossed somehow and thought these were the two you'd suggested

Happy to update to include this one - my bad!

Reply:
Yay! I wanted to make sure that one folks learn about early